File: src/targets/python/python3/fixtures/nested.py
import http.client

conn = http.client.HTTPConnection("mockbin.com")

conn.request("GET", "/har?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value")

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: src/targets/node/fetch/fixtures/nested.js
const fetch = require('node-fetch');

let url = 'http://mockbin.com/har?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value';

let options = {method: 'GET'};

fetch(url, options)
  .then(res => res.json())
  .then(json => console.log(json))
  .catch(err => console.error('error:' + err));